### ISSUE TITLE
Stacked Blocks Are Reset When Island Level Is Recalculated

### ISSUE THREAD

**Issue by octavetoast** [BG-Software-LLC/SuperiorSkyblock2#640]
### Minecraft's Version

Paper 1.17.1

### Plugin's Version

1.8 Stable

### Describe the bug

When running `/is level` or when the level is automatically recalculated, stacked blocks are sometimes reset back to 0. This happens to random islands, not all of them, and there is no clear pattern as to what islands this can happen to.

### To Reproduce

Not sure how to reproduce, other than stacking some blocks and running `/is level`. This doesn't consistently happen, but consistent enough to be destructive.

**Comment by OmerBenGera** [comment]
Are you sure `/is level` is the cause of that?
Maybe they reset before but when doing `/is level` you only see the changes?

**Comment by octavetoast** [comment]
I'm not sure if it's the cause, but it is visually apparent as soon as you run `/is level`. The hologram disappears immediately, and the stack is set back to 0. This isn't a visual bug, the stack is actually 0.

**Comment by OmerBenGera** [comment]
I understand.
Are you able to replicate it on another server?

**Comment by octavetoast** [comment]
I don't have another production server to test it on, if I knew how to reproduce it I could just spin up a local server.

**Comment by OmerBenGera** [comment]
> 
> 
> I don't have another production server to test it on, if I knew how to reproduce it I could just spin up a local server.

You can open a server on your localhost and test it there.

**Comment by octavetoast** [comment]
No, I know. I'm saying I can't replicate the issue consistently which is why I can't test it on another server.

**Comment by OmerBenGera** [comment]
> 
> 
> No, I know. I'm saying I can't replicate the issue consistently which is why I can't test it on another server.

Weird.
Well, try looking into it a bit more and hopefully we'll find what causes it or how to replicate it, so it can be fixed.

**Comment by CryLegende** [comment]
I'm having this bug too, stacked blocks are sometime reset for unknown reason
But I don't know if that really related to the /is level command

**Comment by Cherizer** [comment]
I've had this issue for a while now, still don't know the cause though.

**Comment by nivcoo** [comment]
I've same issue, i have to rollback my db sometimes

**Comment by OmerBenGera** [comment]
No one have yet to provide any information about it. When does it happen, how to replicate the issue, etc.
